src/index.js

```diff
@@ -14,7 +15,13 @@ const port = process.env.PORT || 3001
 
 const app = express()
 
+const corsOptions = {
+  origin: '*',
```

Comment:
Don't allow all origins in a cors configuration. You need to set the exact set of allowed origins to access the information. It should be set to the Frontend's URL. Take a look at the config in other projects